File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/contentRecovery/model/CecoveredComponentInfo.java
package org.ngbp.jsonrpc4jtestharness.rpc.contentRecovery.model;

import java.util.List;

public class CecoveredComponentInfo {
    public List<Component> component;

    class Component {
        public String mediaType;
        public String componentID;
        public String descriptor;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/contentRecovery/model/CecoveredComponentInfo.kt
package org.ngbp.jsonrpc4jtestharness.rpc.contentRecovery.model

import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

data class CecoveredComponentInfo(
        var component: MutableList<Component?>? = null
) : RpcResponse()

data class Component(
        var mediaType: String? = null,
        var componentID: String? = null,
        var descriptor: String? = null
)